gitgud/__main__.py

```diff
@@ -160,6 +162,7 @@ def handle_init(self, args):
         print('Welcome to Git Gud!')
         print()
 
+        init_tracking_json(self.file_operator)
         self.file_operator.get_level().setup(self.file_operator)
         show_tree()
```

Comment:
Change these three lines to use `load_level`. It'll needlessly rewrite the level to the file, but that doesn't change anything, the new code makes the code more maintainable.

`init_tracking_json` will be in the `load_level` function

Reply:
Same as above, yes?